apsuite/asparams.py

```diff
@@ -0,0 +1,35 @@
+"""Nominal Accelerator Structure Parameters."""
+
+# These are nominal accelerator structure parameters conveniently stored
+# in this module. Some parameters are optics-related and should be updated
+# if any lattice changes.
+
+RF_FREQ = 499666000  # [Hz]
+
+BO_RING_LEN = 496.78745  # [m]
```

Comment:
should also depend on `RF_FREQ` and `BO_HARM_NUM`

Reply:
ok, done. 
note: currently the model lattice length matches SI RF frequency at low energy. I had the impression that matching was for high energy...

Reply:
done.